### ISSUE TITLE
error in javascript/angular after login/logout

### ISSUE THREAD

**Issue by svagap** [camptocamp/ngeo#3223]
If user logs out, a error is displayed in the browser-console. The error is repeated, each time a user logs in / out again. Also present on Demo 2.2
```

error message:
error: [$compile:multilink] http://errors.angularjs.org/1.6.5/$compile/multilink
    at desktop.js:13
    at desktop.js:66
    at desktop.js:73
    at d (desktop.js:70)
    at m (desktop.js:74)
    at Object.link (desktop.js:1487)
    at desktop.js:23
    at ua (desktop.js:91)
    at q (desktop.js:77)
    at f (desktop.js:68) "<ngeo-modal ng-model="authCtrl.changePasswordModalShown" class="ng-pristine ng-untouched ng-valid">"
```

**Comment by fredj** [comment]
I think it's fixed with #3211, no error in:
https://camptocamp.github.io/ngeo/master/examples/contribs/gmf/apps/desktop

**Comment by svagap** [comment]
Your right... this can be closed then...
